src/cli/main.rs

```diff
@@ -24,6 +24,13 @@ macro_rules! bail_moved_cmd {
             $src
         )
     };
+    ($src:literal, $dst:literal) => {
```

Comment:
why does it have to be a macro?

Reply:
Just a personal preference here. At some point I believe this code will be deleted anyway, once users have migrated to the new binaries.